tests/wpt/mozilla/tests/css/perspective.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>CSS Test: parsing perspective property</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+
+<body>
+    <div id="perspective0px" style="perspective: 0px"></div>
```

Comment:
Is the issue that setting the style to an invalid value doesn't overwrite the previous value that was there?